get-mods.sh

```diff
@@ -20,7 +20,7 @@ get_special () {
 				REF="$(get_tag)"
 				;;
 			*)
-				if [ -n "$BRANCH" ]; then
+				if [ -n "$BRANCH" ] && git -C dockerweb2-temp show -s --oneline "$BRANCH" 2>/dev/null; then
```

Comment:
Why `2>/dev/null`?

Reply:
To hide stderr output.

Reply:
And what for?

Reply:
To have it not appear in the log because it's useless? It's no hard requirement after all and falls back to the latest tag anyway.

Reply:
Look, I even start bash with -x to get as much transparency as possible and you hide stderr? I opt against.

Reply:
oh, okay, so the following output n times the amount of modules provides an advantage?

```
fatal: ambiguous argument 'feature/database-configuration-backend-3770': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

Reply:
How many lines are there **relative to the other printed stuff**?

Reply:
26 lines on average, 3 additional ones, that's an increase by 13%.

Reply:
Oh and

> That's not the point, nor answers my question.

Reply:
> an increase by 13%

Which is still OK for me.